pygtrie.py

```diff
@@ -1223,6 +1223,16 @@ def __init__(self, *args, **kwargs):
         self._separator = kwargs.pop('separator', '/')
         super(StringTrie, self).__init__(*args, **kwargs)
 
+    @property
+    def _separator(self):
+        return self.__separator
```

Comment:
I don’t mind private fields. But I’m not sure why this needs to be done via a getter and setter. Why not just add a check in `__init__`?